### ISSUE TITLE
Not working on Polygon

### ISSUE THREAD

**Issue by saranonearth** [curvefi/curve-js#13]
```js
(async () => {

    await curve.default.init('JsonRpc', {
        url: "https://polygon-mainnet.g.alchemy.com/v2/tY30vNwB3HhGJIikCtOvL7-Mo-_6f6CY",
        privateKey
    },
    {
        chainId: 137
    });

})();
```
Trying to use curve-js on polygon chain. Getting the following error. Any input would be of great help.
```
CURVE-JS IS CONNECTED TO NETWORK: {
  name: 'matic',
  chainId: 137,
  ensAddress: null,
  _defaultProvider: null
}
E:\dev\curve\node_modules\@ethersproject\logger\lib\index.js:199
        var error = new Error(message);
                    ^

Error: resolver or addr is not configured for ENS name (argument="name", value="", code=INVALID_ARGUMENT, version=contracts/5.5.0)
    at Logger.makeError (E:\dev\curve\node_modules\@ethersproject\logger\lib\index.js:199:21)
    at Logger.throwError (E:\dev\curve\node_modules\@ethersproject\logger\lib\index.js:208:20)
    at Logger.throwArgumentError (E:\dev\curve\node_modules\@ethersproject\logger\lib\index.js:211:21)
    at E:\dev\curve\node_modules\@ethersproject\contracts\lib\index.js:109:32
    at step (E:\dev\curve\node_modules\@ethersproject\contracts\lib\index.js:48:23)
    at Object.next (E:\dev\curve\node_modules\@ethersproject\contracts\lib\index.js:29:53)
    at fulfilled (E:\dev\curve\node_modules\@ethersproject\contracts\lib\index.js:20:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  reason: 'resolver or addr is not configured for ENS name',
  code: 'INVALID_ARGUMENT',
  argument: 'name',
  value: ''
}
```

**Comment by Macket** [comment]
Will fix asap. Thanks!

**Comment by Macket** [comment]
Fixed by f26b3c92386e37e890df96b0a2fce650db575478
